src/credentials/attestation_verifier/TestDACRevocationDelegateImpl.cpp

```diff
@@ -116,14 +213,32 @@ bool TestDACRevocationDelegateImpl::IsEntryInRevocationSet(const CharSpan & akid
     return false;
 }
 
-CHIP_ERROR TestDACRevocationDelegateImpl::GetAKIDHexStr(const ByteSpan & certDer, MutableCharSpan & outAKIDHexStr)
+CHIP_ERROR TestDACRevocationDelegateImpl::GetKeyIDHexStr(const ByteSpan & certDer, MutableCharSpan & outKeyIDHexStr, bool isAKID)
 {
-    uint8_t akidBuf[kAuthorityKeyIdentifierLength];
-    MutableByteSpan akid(akidBuf);
+    // kAuthorityKeyIdentifierLength and kSubjectKeyIdentifierLength are equal
```

Comment:
That should be a static_assert, not a comment.

Reply:
thanks, added the static_assert